### ISSUE TITLE
problem with docker 1.12.1

### ISSUE THREAD

**Issue by mschneider82** [FalconSocial/berth#9]
```
» berth -v -d sample.yaml
>>> Reading the configurationsamplefile.
>> The configuration file "sample.yaml" has been parsed.
>>> Parsed configuration:
{'environment': {'VERSION': '3.8.0'}, 'build': {'image': 'busybox', 'volumes': {'.': '/data'}, 'script': '#!/bin/sh -ex\n\ntouch /data/build-file-v${VERSION}\necho "Hello, containerized World!"\necho "Error, from the container!" >&2\n'}, 'package': {'fpm': {'maintainer': 'Falcon Social', 'description': 'Example of a package description', 'package': '/data/sample-package_${VERSION}.deb', 'url': 'https://github.com/FalconSocial/berth', 'iteration': 1, 'name': 'sample-package', 's': 'dir', 'version': '$VERSION', 'architecture': 'amd64', '_arguments': {'/data/build-file-v${VERSION}': '/build-file'}, 't': 'deb'}, 'image': 'tenzer/fpm', 'volumes': {'.': '/data'}}}
>> The configuration has been verified without errors.
>>> Checking if "busybox:latest" is among the local images in Docker.
>> Docker image found.
> Building project.
>>> Writing build script to temporary file.
>> Wrote temporary build script to "/home/user/git/sampledir/berth-build-vD6GzQ.sh".
>>> Starting build container.
Traceback (most recent call last):
  File "/home/user/git/sampledir/venv/bin/berth", line 11, in <module>
    sys.exit(main())
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/click/core.py", line 700, in __call__
    return self.main(*args, **kwargs)
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/click/core.py", line 680, in main
    rv = self.invoke(ctx)
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/click/core.py", line 873, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/click/core.py", line 508, in invoke
    return callback(*args, **kwargs)
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/click/decorators.py", line 16, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/berth/cli.py", line 48, in main
    if not build.build(configuration):
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/berth/build.py", line 37, in build
    docker.start(container['Id'], binds=binds)
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/docker/utils/decorators.py", line 21, in wrapped
    return f(self, resource_id, *args, **kwargs)
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/docker/api/container.py", line 383, in start
    self._raise_for_status(res)
  File "/home/user/git/sampledir/venv/local/lib/python2.7/site-packages/docker/client.py", line 167, in _raise_for_status
    raise errors.APIError(e, response, explanation=explanation)
docker.errors.APIError: 400 Client Error: Bad Request ("{"message":"starting container with HostConfig was deprecated since v1.10 and removed in v1.12"}")

```

**Comment by areian** [comment]
Fixed in #10

**Comment by areian** [comment]
Until this is merged, you can fix the error by downgrading docker-py to version 1.9.0

```
pip install docker-py==1.9.0
```

**Comment by areian** [comment]
Fixed in v1.3.5
